README.md

```diff
@@ -96,7 +96,17 @@ Assuming you haven’t created any functions yet, the response should be:
 }
 ```
 
-## Step 5: Build, package, and deploy
+## Step 5: Create a bucket to store package
+
+This creates a Stedi Bucket that will keep your code.
+
+```console
+uuid=$(uuidgen)
+export BUCKET_NAME="cli-example-${uuid:l}"
+npx stedi buckets create-bucket --bucket-name ${BUCKET_NAME}
+```
```

Comment:
This is neat. However, it’s a bit too much magic for my taste. Either we should explain it, or we should let the user take care of the name manually. I prefer the latter for a tutorial, because that gives them a better idea of what’s going on.

Reply:
alright. i will export the environment variable with `<YOUR_UNIQUE_NAME_HERE>`

Reply:
I also made some changes, but I don’t use an environment variable at all. We can still do that, if you think it’s clearer.

Reply:
my mistake. i didn't realize you pushed into my branch. may i suggest you to use the [suggest change feature from github](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request#starting-a-review). the only time it does not work is when you need to suggest in a file that has not been modified in the PR.

Reply:
i think it is best to follow whatever approach it is being used in the rest of the documentation in order to make it more consistent. we have an similar situation here with `<YOUR-USER-OR-ORG>`: https://www.stedi.com/docs/tutorials/stedi-scheduled-sftp-poller-demo#prerequisites

Reply:
i agree on removing the environment variable approach so that it is more explicit in each command you are running.

Reply:
i pushed a change with details about the UUID having to be lowercase and adding the angle brackets around the bucket name.

Reply:
Sorry about that. I actually didn’t now that you could leave suggestions. 😲  I’ll use that from now on.

Reply:
We still don’t have a single approach for INSERT YOUR VALUE HERE. Partly because different people write it differently, mostly because I have yet to pick one and enforce it. 😬